topeft/params/SMEFTsim-topU3l_dim6top.yml

```diff
@@ -0,0 +1,23 @@
+# Weinberg angle for rotating ctW,ctB <-> ctW,ctZ (assuming cHWB=0)
+"CW": 0.88136 # PDG W / Z mass ratio 
+"SW": "np.sqrt(1 - np.square({CW}))" # eval in python
+
+# SMEFTsim j <-> dim6top q
+"ctj1": "ctq1"
+"cQj31": "cQq13"
+"ctj8": "ctq8"
+"cQj11": "cQq11"
+"cQj18": "cQq81"
+"cQj38": "cQq83"
+
+"cQQ1": ["expr::SEMFTsim_cQQ1('0.5 * @0', {cQQ1})", ["cQQ1"]]
+"ctt": "ctt1"
+
+# ctG relative minus sign
+"ctGRe": ["expr::ctGRe('-1 * @0', {ctG})", ["ctG"]]
+"ctGIm": ["expr::ctGIm('-1 * @0', {ctG})", ["ctG"]]
```

Comment:
Does this mean that `ctGRe` and `ctGIm` will be scaled/set/controlled by a single parameter (e.g. 'ctG')? Do we not assume the imaginary part for this (and other) operators is 0?

Reply:
Oh that was a mistake, it should be ctGI for ctGIm